library/ui-styles/src/main/res/values/styles_voice_message.xml

```diff
@@ -0,0 +1,154 @@
+        <item name="android:src">@drawable/circle</item>
+        <item name="layout_goneMarginStart">24dp</item>
+        <item name="tint">@color/palette_vermilion</item>
+    </style>
+
+    <style name="VoicePlaybackControlButton">
+        <item name="android:layout_width">32dp</item>
+        <item name="android:layout_height">32dp</item>
+        <item name="android:background">@drawable/bg_voice_play_pause_button</item>
+        <item name="android:paddingStart">3dp</item>
+        <item name="android:paddingEnd">0dp</item>
+        <item name="android:layout_marginStart">8dp</item>
+        <item name="android:src">@drawable/ic_voice_play</item>
+        <item name="android:backgroundTint">?vctr_voice_message_play_pause_button_background</item>
+    </style>
+
+    <style name="VoicePlaybackTime" parent="VoiceMessageTimer">
+        <item name="android:layout_marginStart">6dp</item>
+        <item name="layout_goneMarginStart">24dp</item>
+    </style>
+
+    <style name="VoicePlaybackWaveform">
+        <item name="android:layout_width">0dp</item>
+        <item name="android:layout_height">0dp</item>
+        <item name="android:layout_marginStart">8dp</item>
+        <item name="android:layout_marginEnd">8dp</item>
+        <item name="chunkColor">@color/palette_gray_300</item>
+        <item name="chunkAlignTo">center</item>
+        <item name="chunkMinHeight">1dp</item>
+        <item name="chunkRoundedCorners">true</item>
+        <item name="chunkSoftTransition">true</item>
+        <item name="chunkSpace">2dp</item>
+        <item name="chunkWidth">2dp</item>
+        <item name="direction">leftToRight</item>
+    </style>
+
+    <style name="VoiceTimelinePlaybackLayout">
+        <item name="android:layout_width">0dp</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:background">@drawable/bg_voice_playback</item>
+        <item name="android:minHeight">48dp</item>
+        <item name="android:paddingStart">8dp</item>
+        <item name="android:paddingTop">6dp</item>
+        <item name="android:paddingEnd">8dp</item>
+        <item name="android:paddingBottom">6dp</item>
+        <item name="android:backgroundTint">?vctr_voice_message_playback_background</item>
+    </style>
+
+    <style name="VoiceTimelinePlaybackControlButton">
+        <item name="android:layout_width">32dp</item>
+        <item name="android:layout_height">32dp</item>
+        <item name="android:background">@drawable/bg_voice_play_pause_button</item>
+        <item name="android:paddingStart">3dp</item>
+        <item name="android:paddingEnd">0dp</item>
+        <item name="android:src">@drawable/ic_voice_play</item>
+        <item name="android:backgroundTint">?vctr_voice_message_play_pause_button_background</item>
+    </style>
+
+    <style name="VoiceTimelinePlaybackTime" parent="VoiceMessageTimer">
+        <item name="android:layout_marginStart">6dp</item>
+    </style>
```

Comment:
All attributes with `layout_` should be in the layout file. Lets keep usage of style only for the themes (colors, font, etc.)

Reply:
Done